payments-core/src/main/java/com/stripe/android/payments/core/injection/AuthenticationComponent.kt

```diff
@@ -31,6 +34,8 @@ import kotlin.coroutines.CoroutineContext
 internal interface AuthenticationComponent {
     val registry: DefaultPaymentAuthenticatorRegistry
 
+    fun inject(stripe3ds2TransactionViewModel: Stripe3ds2TransactionViewModel)
```

Comment:
we can define other methods like the following to inject into `Activity`s
`fun inject(stripe3ds2TransactionActivity: Stripe3ds2TransactionActivity)`